### PR TITLE
adds belledejour content

### LINKED ISSUE #12: Project 7: Belledejour (photography articles)
image

### PR DESCRIPTION
**Pull request by sssoja**
### What changes have you made?

- added content to the belledejour page 
- Aitor mentioned maybe using a video but for now let's leave it here and see if he's happy with it
- all images have been compressed

### Which issue(s) does this PR fix?

Fixes #12

### Screenshots (if there are design changes)
Screenshot 2021-01-08 at 16 11 08

### How to test

- open the browser on http://localhost:3000/projects/belledejour
- move the timer back to view the page content

### CODE REVIEW

File: src/components/projects/BelledejourContent.tsx [hunk 1]
```diff
@@ -0,0 +1,135 @@
+import { LayoutProps, layout } from "styled-system";
+
+import Grid from "../Grid";
+import React from "react";
+import image1 from "./../../assets/project-page/belledejour/belledejour-1.jpg";
```
Comment (Jbarget) [inline src/components/projects/BelledejourContent.tsx:5]: these import statements theres a minor change:
`"./../../" === "../../"`

**Review by Jbarget — APPROVED** [review]
After making the changes to all the imports this is good to go in

File: src/components/projects/BelledejourContent.tsx (hunk 1 above)
Comment (sssoja) [reply src/components/projects/BelledejourContent.tsx:5]: hmmmmmm

Comment (sssoja) [reply src/components/projects/BelledejourContent.tsx:5]: I've done this throughout the codebase - lemme put in a ticket to get rid

**Pull request merged**